qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -151,6 +151,22 @@
         - driver_update:
             only all_drivers
             type = win_virtio_driver_update_by_installer
+            basic_io_test = yes
```

Comment:
Hi, I think it's better to moving this part to the common place, such as before L43.
And only left key parameters which is  used to start the device in qemu kvm cmd line,such as "filesystems"